dotnet/src/dotnetcore/DynService/Cosmos/CosmosDBDataReader.cs

```diff
@@ -189,9 +189,9 @@ public string GetDataTypeName(int i)
 
 		public DateTime GetDateTime(int i)
 		{
-		   if (GetAttValue(i) is DateTime value)
-				return value;
-			return default(DateTime);
+			DateTime.TryParseExact(GetAttValue(i).ToString(), CosmosDBHelper.ISO_DATE_FORMAT, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal, out DateTime dt);
```

Comment:
GetAttValue(i).ToString() is using the current culture to create a string. It should specify the culture to get the same result in any configuration.